xhr/client/src/main/scala/endpoints4s/xhr/EndpointsSettings.scala

```diff
@@ -0,0 +1,37 @@
+package endpoints4s.xhr
+
+import endpoints4s.Hashing
+
+import scala.annotation.nowarn
+
+final class EndpointsSettings(val host: Option[String]) extends Serializable {
+
+  override def toString =
+    s"EndpointsSettings($host)"
+
+  override def equals(other: Any): Boolean =
+    other match {
+      case that: EndpointsSettings =>
+        host == that.host
+      case _ => false
+    }
+
+  override def hashCode(): Int = Hashing.hash(host)
+
+  @nowarn("cat=unused")
```

Comment:
It is used to implement `withHost`. Is this a leftover?

Reply:
I copy pasted this from Fetch implementation, IIRC this was required for Scala 3 support.